drivers/watchdog/wdt_mcux_wwdt.c

```diff
@@ -25,6 +25,7 @@ LOG_MODULE_REGISTER(wdt_mcux_wwdt);
 struct mcux_wwdt_config {
 	WWDT_Type *base;
 	uint8_t clk_divider;
+	uint8_t wdt_instance;
```

Comment:
if using the clock control api, this isnt needed

Reply:
No longer applicable.